packages/tradershub/src/features/cfd/flows/CTrader/AvailableCTraderAccountsList/AvailableCTraderAccountsList.tsx

```diff
@@ -8,10 +9,29 @@ import {
 import { getStaticUrl } from '../../../../../helpers/urls';
 import CTrader from '../../../../../public/images/cfd/ctrader.svg';
```

Comment:
Can we use it from deriv/quill-icons

It is available there I think

Reply:
Yes yes i have another PR for that

Reply:
Provide the link here so i can resolve this

Reply:
https://github.com/binary-com/deriv-app/pull/12989

Reply:
@hamza-deriv as we discussed with @niloofar-deriv yesterday, there is a performance issue for quill-icons which prevent us to get it from different directories, for example `@deriv/quill-icons/dist/Currencies`. She will notify us once it's fixed.